src/Doctrine/Version.php

```diff
@@ -17,15 +17,19 @@ class Version
     /**
      * We're g̶u̶e̶s̶s̶i̶n̶g̶ doing empirical research on which versions of SQLite
      * support JSON. So far, tests indicate:
+     * https://www.sqlite.org/json1.html --> JSON since SQLite version 3.9.0 (2015-10-14)
+     * - 3.32.2 - OK (Wytse's FBSD 12.1 \w PHP 7.2)
      * - 3.20.1 - Not OK (Travis PHP 7.2)
      * - 3.27.2 - OK (Bob's Raspberry Pi, running PHP 7.3.11 on Raspbian)
      * - 3.28.0 - OK (Travis PHP 7.3)
      * - 3.28.0 - Not OK (Bob's PHP 7.2, installed with Brew)
      * - 3.29.0 - OK (MacOS Mojave)
      * - 3.30.1 - OK (MacOS Catalina)
      */
-    public const SQLITE_WITH_JSON = '3.27.0';
-    public const PHP_WITH_SQLITE = '7.3.0';
+    // JSON supported since SQLite version 3.9.0
+    public const SQLITE_WITH_JSON = '3.9.0';
+    // PHP supports SQLite since version 5.3.0
+    public const PHP_WITH_SQLITE = '5.3.0';
```

Comment:
This is not correct. If we merge this in, Bolt will absolutely assume it has JSON support on installations where it doesn't have it. 

Maybe we should change the name of the constant, because it's more "PHP_WITH_SQLITE_THAT_HAS_JSON_SUPPORT_BUNDLED". I've definitely seen this break on PHP 7.2 setups. (Ubuntu 2018.04, if i recall correctly)

Ideally, we could properly test if we have it, but I have not found a proper way to do that.. If you have a good suggestion, I'm all for it!

Reply:
I've written a small test and implemented in a new version (e827a799). Meanwhile I'll leave the (restored) old code commented out until we've fully tested everything. Cleanup is the last step.